x/clp/keeper/calculations.go

```diff
@@ -277,16 +359,32 @@ func ReducePrecision(dec sdk.Dec, po int64) sdk.Dec {
 	return dec.Quo(p)
 }
 
+func ReducePrecisionInt(num sdk.Int, po int64) sdk.Int {
+	multiplier := big.NewInt(10).Exp(big.NewInt(10), big.NewInt(po), nil)
+	multiplierInt := sdk.NewIntFromBigInt(multiplier)
+	fmt.Println(num.String(), multiplierInt.String())
+	val := num.Quo(multiplierInt)
+	fmt.Println("val", val)
```

Comment:
Is there a reason why we're sending this to STDOUT? Debug?

Reply:
yeah that needs to be cleaned up.

Reply:
On it